tests/admin_scripts/tests.py

```diff
@@ -1822,6 +1844,50 @@ def test_label_command_multiple_label(self):
             "('settings', None), ('traceback', False), ('verbosity', 1)]"
         )
 
+    def test_with_suppress_command_help(self):
+
+        args = ['with_suppress_command', '--help']
+        out, err = self.run_manage(args)
+        self.assertNoOutput(err)
+        self.assertOutput(out, 'Test basic commands with suppress')
+        self.assertOutput(out, '--help')
+        self.assertNotInOutput(out, '--verbosity')
+        self.assertNotInOutput(out, '--traceback')
+        self.assertNotInOutput(out, '--pythonpath')
+        self.assertNotInOutput(out, '--settings')
+        self.assertNotInOutput(out, '--version')
+        self.assertNotInOutput(out, '--no-color')
+        self.assertNotInOutput(out, '--force-color')
+
+    def test_with_suppress_command_defaults(self):
+        args = ['with_suppress_command', 'testLabel']
+        out, err = self.run_manage(args)
+        self.assertNoOutput(err)
+        self.assertOutput(
+            out,
+            "EXECUTE:WithSuppressCommand labels=('testLabel',), options=[('force_color', False), "
+            "('no_color', False), ('pythonpath', None), ('settings', None), ('traceback', False), ('verbosity', 1)]"
+        )
+
+    def test_add_base_argument(self):
+        args = ['base_command', '--help']
+        out, err = self.run_manage(args)
+        self.assertNoOutput(err)
+        self.assertOutput(
+            out,
+            "usage: manage.py base_command [-h] [--option_a OPTION_A] [--option_b OPTION_B]\n"
+            "                              [--option_c OPTION_C] [--version]\n"
+            "                              [--verbosity {0,1,2,3}] [--settings SETTINGS]\n"
+            "                              [--pythonpath PYTHONPATH] [--traceback]\n"
+            "                              [--no-color] [--force-color]\n"
+            "                              [args [args ...]]"
+        )
+        self.assertOutput(
+            out,
+            "--verbosity {0,1,2,3}, -v {0,1,2,3}"
+        )
```

Comment:
There is no need to check the entire output, we can check the list of expected options, e.g.
```suggestion
        expected_options = [
            '-h',
            '--option_a OPTION_A',
            '--option_b OPTION_B',
            '--option_c OPTION_C',
            '--version',
            '--verbosity {0,1,2,3}',
            '--settings SETTINGS',
            '--pythonpath PYTHONPATH',
            '--traceback',
            '--no-color',
            '--force-color',
        ]
        for option in expected_options:
            self.assertOutput(out, f'[{option}]')
```

Reply:
Yes, but I also wanted to make sure that default values are passed despite suppressed help.
Anyway, It is easier to use your approach.